### PR TITLE
Timelapse Update -3 

### PR DESCRIPTION
**Pull request by rachelcynthia**
This has both front end and back end for the web page with JS

### CODE REVIEW

File: timelapse.html [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+        <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
+
+<!-- Optional theme -->
+<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap-theme.min.css" integrity="sha384-rHyoN1iRsVXV4nD0JutlnGaslCJuC7uwjduW9SVrLvRYooPp2bWYgmgJQIXwl/Sp" crossorigin="anonymous">
+
+<!-- Latest compiled and minified JavaScript -->
+<script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js" integrity="sha384-Tc5IQib027qvyjSMfHjOMaLkfuWVxZxUPnCJA7l2mCWNIpG9mGCD8wGNIcPD7Txa" crossorigin="anonymous"></script>
+        <link rel="stylesheet" href="index.css" type="text/css">
+        <script>
+            window.onload=today;
```
Comment (bhavaniravi) [inline timelapse.html:13]: move js code to a separate file

Comment (bhavaniravi) [reply timelapse.html:13]: use a linter and format the JS code

File: index.css [hunk 2]
```diff
@@ -0,0 +1,93 @@
+#banner_image{
+    padding-top: 75px;
+    padding-bottom: 50px;
+    text-align: center;
+    color:black;
+    background-color:rgba(0,0,0,0.7);
+    background: url("images.jpg") no-repeat center center;
```
Comment (bhavaniravi) [inline index.css:7]: Remove the background image, it's okay for it to be white

File: timelapse.html [hunk 3]
```diff
@@ -0,0 +1,178 @@
+<html>
+    <head>
+        <title></title>
+        <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
+
+<!-- Optional theme -->
+<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap-theme.min.css" integrity="sha384-rHyoN1iRsVXV4nD0JutlnGaslCJuC7uwjduW9SVrLvRYooPp2bWYgmgJQIXwl/Sp" crossorigin="anonymous">
+
+<!-- Latest compiled and minified JavaScript -->
+<script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js" integrity="sha384-Tc5IQib027qvyjSMfHjOMaLkfuWVxZxUPnCJA7l2mCWNIpG9mGCD8wGNIcPD7Txa" crossorigin="anonymous"></script>
```
Comment (bhavaniravi) [inline timelapse.html:10]: I advice not to use bootstrap and try out css-grid, it's equally easy and not heavy.

File: timelapse.html [hunk 4]
```diff
@@ -0,0 +1,178 @@
...
+                <br>
+                <div id="output3.2">
+                    Ans
+                </div>
+                </div>
+            </div>
+            <br><br>
+       <footer>
+        <div>
+            <div class="col-xs-6">
```
Comment (bhavaniravi) [inline timelapse.html:152]: Remove this section

File: timelapse.html [hunk 5]
```diff
@@ -0,0 +1,178 @@
...
+                        <span class="icon-bar"></span>
+                        <span class="icon-bar"></span>
+                    </button>
+                    <center>
+                    <div class="navbar-brand">Timelapse</div>
+                    </center>
+                </div>
+            </div>
+        </div>
+        <div id="banner_image">
```
Comment (bhavaniravi) [inline timelapse.html:100]: Make this a navbar element

File: timelapse.html [hunk 6]
```diff
@@ -0,0 +1,178 @@
...
+            <hr>
+            <div class="container container1" id="task1">
+                <div class="col-xs-6">
+                    <input type="date" id="mydate" class="my_date" value="">
+                    &nbsp;&nbsp;&nbsp;&nbsp;
+                    <span>&#43;</span>
+                    &nbsp;&nbsp;&nbsp;&nbsp;
+                    <input type="number" id="days" onchange="new_date()" placeholder="Number of days">
+                    &nbsp;&nbsp;&nbsp;&nbsp;
+                    <span>&equals;</span>
```
Comment (bhavaniravi) [inline timelapse.html:113]: = should be a button, it's not intiutive

File: timelapse.html [hunk 7]
```diff
@@ -0,0 +1,178 @@
...
+            <br><hr>
+            <div class="container container1" id="task2">
+                <div class="col-xs-2">From</div>
+                <div class="col-xs-2"></div>
+                <div class="col-xs-2">To</div>
+                <div class="col-xs-6"></div>
+                <br>
+                <div>
+                <div class="col-xs-2"><input type="date" id="from_date" class="my_date" value="" onchange="get_days()"></div>
+                <div class="col-xs-2"></div>
```
Comment (bhavaniravi) [inline timelapse.html:127]: I really like how the result changes as you click on the dates.

**Comment by rachelcynthia** [comment]
We have updated the timelapse web page with the changes

**Comment by rachelcynthia** [comment]
The react components are added to the webpage.

**Pull request closed without merging**